D3D12HelloWorld/Graphics/PipelineState.cs

```diff
@@ -47,9 +53,8 @@ public PipelineState(GraphicsDevice device, ID3D12RootSignature rootSignature, I
             var cullNone = RasterizerDescription.CullNone;
             cullNone.FrontCounterClockwise = true;
             var alphaBlend = BlendDescription.AlphaBlend;
-            var inputLayout = new InputLayoutDescription(inputElements.Select((InputElementDescription i) => Unsafe.As<InputElementDescription, Vortice.Direct3D12.InputElementDescription>(ref i)).ToArray());
             var graphicsPipelineStateDescription = new GraphicsPipelineStateDescription {
-                InputLayout = inputLayout,
+                InputLayout = new InputLayoutDescription(inputElements),
```

Comment:
Unlike the original DX12GE, I'm not using their types to mirror the Vortice ones, so no need for the call to `Unsafe.As<>`